### PR TITLE
Release v0.22.0

### PR DESCRIPTION
**Pull request by plafer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+
+This release includes major improvements in making the library compatible with ibc-go v5.0.1. This includes making ibc events compatible and removing the crossing-hellos logic from the connection and channel handshakes.
+
+### BREAKING CHANGES
+
+- Make connection events compatible with ibc-go
+  ([#145](https://github.com/cosmos/ibc-rs/issues/145))
+- Makes channel/packet events compatible with ibc-go
+  ([#146](https://github.com/cosmos/ibc-rs/issues/146))
+- Remove crossing hellos logic from channel handshake
```
Comment (hu55a1n1) [inline CHANGELOG.md:18]: Looks like we're missing an entry for #156 here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -222,16 +222,18 @@ Our release process is as follows:
...
       in this release.
+      1. Your text editor will open. Write the release summary, and close the editor.
+      2. Add this same summary to `CHANGELOG.md` as well.
    3. Committing the updated `CHANGELOG.md` file and `.changelog` directory to the repo.
 2. Push this to a branch `release/vX.Y.Z` according to the version number of
    the anticipated release (e.g. `release/v0.18.0`) and open a **draft PR**.
 3. Bump all relevant versions in the codebase to the new version and push these
    changes to the release PR. This includes:
-   1. All `Cargo.toml` files (making sure dependencies' versions are updated
+   1. `crates/ibc/Cargo.toml` files (making sure dependencies' versions are updated
```
Comment (hu55a1n1) [inline CONTRIBUTING.md]:
```suggestion
   1. `crates/ibc/Cargo.toml` file (making sure dependencies' versions are updated
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -222,16 +222,18 @@ Our release process is as follows:
...
       too).
-   2. All crates' `lib.rs` files documentation references' `html_root_url`
+   2. `crates/ibc/lib.rs` files documentation references' `html_root_url`
```
Comment (hu55a1n1) [inline CONTRIBUTING.md]:
```suggestion
   2. `crates/ibc/lib.rs` file documentation references' `html_root_url`
```

**Review by hu55a1n1 — APPROVED** [review]
Suggested some minor changes. (preapproved)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+  ([#157](https://github.com/cosmos/ibc-rs/issues/157))
+- call validate_self_client in conn_open_try and conn_open_ack,
+  and provide a tendermint implementation for validate_self_client
+  ([#162](https://github.com/cosmos/ibc-rs/issues/162))
+- Refactor channel handlers. Proof calls were inlined, and our handshake
+  variable naming convention was applied ([#166](https://github.com/cosmos/ibc-
+  rs/issues/166))
+- Change ClientType to contain a String instead of &'static str
```
Comment (romac) [inline CHANGELOG.md]:
```suggestion
- Change `ClientType` to contain a `String` instead of `&'static str`
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+- Makes channel/packet events compatible with ibc-go
+  ([#146](https://github.com/cosmos/ibc-rs/issues/146))
+- Remove crossing hellos logic from channel handshake
+  ([#157](https://github.com/cosmos/ibc-rs/issues/157))
+- call validate_self_client in conn_open_try and conn_open_ack,
+  and provide a tendermint implementation for validate_self_client
+  ([#162](https://github.com/cosmos/ibc-rs/issues/162))
+- Refactor channel handlers. Proof calls were inlined, and our handshake
+  variable naming convention was applied ([#166](https://github.com/cosmos/ibc-
+  rs/issues/166))
```
Comment (romac) [inline CHANGELOG.md]:
```suggestion
  variable naming convention was applied
  ([#166](https://github.com/cosmos/ibc-rs/issues/166))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+### BREAKING CHANGES
+
+- Make connection events compatible with ibc-go
+  ([#145](https://github.com/cosmos/ibc-rs/issues/145))
+- Makes channel/packet events compatible with ibc-go
+  ([#146](https://github.com/cosmos/ibc-rs/issues/146))
+- Remove crossing hellos logic from channel handshake
+  ([#157](https://github.com/cosmos/ibc-rs/issues/157))
+- call validate_self_client in conn_open_try and conn_open_ack,
+  and provide a tendermint implementation for validate_self_client
```
Comment (romac) [inline CHANGELOG.md]:
```suggestion
  and provide a tendermint implementation for `validate_self_client`
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,43 @@
...
+
+### BREAKING CHANGES
+
+- Make connection events compatible with ibc-go
+  ([#145](https://github.com/cosmos/ibc-rs/issues/145))
+- Makes channel/packet events compatible with ibc-go
+  ([#146](https://github.com/cosmos/ibc-rs/issues/146))
+- Remove crossing hellos logic from channel handshake
+  ([#157](https://github.com/cosmos/ibc-rs/issues/157))
+- call validate_self_client in conn_open_try and conn_open_ack,
```
Comment (romac) [inline CHANGELOG.md]:
```suggestion
- call `validate_self_client` in `conn_open_try` and `conn_open_ack`,
```

File: CHANGELOG.md (hunk 1 above)
Comment (plafer) [reply CHANGELOG.md:18]: [c16862d](https://github.com/cosmos/ibc-rs/pull/234/commits/c16862defda47ad9af214726167b95e4542f1dd1)

File: CHANGELOG.md (hunk 5 above)
Comment (plafer) [reply CHANGELOG.md]: good catch!

**Pull request merged**